Formula/pdftilecut.rb

```diff
@@ -0,0 +1,25 @@
+class Pdftilecut < Formula
+  desc "Sub-divide a PDF page(s) into smaller pages so you can print them"
+  homepage "https://github.com/oxplot/pdftilecut"
+  url "https://github.com/oxplot/pdftilecut.git",
+      tag: "v0.4",
+      revision: "af9ca3bff60eed47026a6e1c40b6ef85e5b36393"
```

Comment:
```suggestion
      tag:      "v0.4",
      revision: "af9ca3bff60eed47026a6e1c40b6ef85e5b36393"
```
Sorry, forgot about this one

Reply:
No problem, Thanks !